sample/gradle.properties

```diff
@@ -0,0 +1 @@
+autodispose.classes_with_scope=android.app.Activity,android.app.Fragment,com.uber.autodispose.LifecycleScopeProvider,android.arch.lifecycle.LifecycleOwner,androidx.lifecycle.LifecycleOwner,com.uber.autodispose.ScopeProvider,com.uber.autodispose.sample.CustomScope
```

Comment:
No need for the android ones since lint is additive and adds android ones by default.

Reply:
6d08b33